docs/content/3.manual/manual.yml

```diff
@@ -1631,12 +1639,16 @@ sections:
       - title: "`..`"
```

Comment:
This is now the only operator without a name. How about `Recurse operator` or `Expansion operator`?